priv/templates/nimble_template/test/otp_app_web/helpers/icon_helper_test.exs.eex

```diff
@@ -0,0 +1,19 @@
+defmodule <%= web_module %>.IconHelperTest do
+  use <%= web_module %>.ConnCase, async: true
+
+  import Phoenix.HTML, only: [safe_to_string: 1]
+
+  alias <%= web_module %>.IconHelper
+
+  describe "icon_tag/2" do
```

Comment:
Should it be
```suggestion
  describe "icon_tag/3" do
```

Reply:
updated in https://github.com/nimblehq/elixir-templates/pull/146/commits/5db504f70b06726df8efe19e74b4abd1e173611f @rosle